gprofiler/profilers/perf.py

```diff
@@ -80,7 +80,7 @@ def start(self) -> None:
         except TimeoutError:
             process.kill()
             assert process.stdout is not None and process.stderr is not None
-            logger.error(f"perf failed to start. stdout {process.stdout.read()!r} stderr {process.stderr.read()!r}")
+            logger.critical(f"perf failed to start. stdout {process.stdout.read()!r} stderr {process.stderr.read()!r}")
```

Comment:
Same for this one, to be honest: https://github.com/Granulate/gprofiler/pull/388/files#r906863849

Reply:
Done

Reply:
Although I didn't exit brutally, that's a decision for another PR